crates/nu-command/src/platform/input/list.rs

```diff
@@ -255,6 +249,84 @@ impl Command for InputList {
     }
 }
 
+use dialoguer::theme::{SimpleTheme, Theme};
+use nu_ansi_term::ansi::RESET;
+
+// could potentially be used to map the use theme colors at some point
+
+/// Theme for handling already colored items gracefully.
+struct NuTheme;
+
+impl Theme for NuTheme {
+    fn format_select_prompt_item(
+        &self,
+        f: &mut dyn std::fmt::Write,
+        text: &str,
+        active: bool,
+    ) -> std::fmt::Result {
+        SimpleTheme.format_select_prompt_item(f, text, active)?;
+        write!(f, "{RESET}")
+    }
+
+    fn format_multi_select_prompt_item(
+        &self,
+        f: &mut dyn std::fmt::Write,
+        text: &str,
+        checked: bool,
+        active: bool,
+    ) -> std::fmt::Result {
+        SimpleTheme.format_multi_select_prompt_item(f, text, checked, active)?;
+        write!(f, "{RESET}")
+    }
+
+    fn format_sort_prompt_item(
+        &self,
+        f: &mut dyn std::fmt::Write,
+        text: &str,
+        picked: bool,
+        active: bool,
+    ) -> std::fmt::Result {
+        SimpleTheme.format_sort_prompt_item(f, text, picked, active)?;
+        writeln!(f, "{RESET}")
+    }
+
+    fn format_fuzzy_select_prompt_item(
+        &self,
+        f: &mut dyn std::fmt::Write,
+        text: &str,
+        active: bool,
+        highlight_matches: bool,
+        matcher: &fuzzy_matcher::skim::SkimMatcherV2,
+        search_term: &str,
+    ) -> std::fmt::Result {
+        use fuzzy_matcher::FuzzyMatcher;
+        write!(f, "{} ", if active { ">" } else { " " })?;
+
+        if !highlight_matches {
+            return write!(f, "{text}{RESET}");
+        }
+        let Some((_score, indices)) = matcher.fuzzy_indices(text, search_term) else {
+            return write!(f, "{text}{RESET}");
+        };
+        let prefix = nu_ansi_term::Style::new()
+            .bold()
+            .italic()
+            .prefix()
+            .to_string();
+        // HACK: Reset bold italic, from the `ansi` command, should be moved to `nu_ansi_term`
+        let suffix = "\x1b[22;23m";
```

Comment:
The italication is a bit subtle but I get the impetus if we want to gracefully handle colors.